pylds/base.py

```diff
@@ -186,18 +286,32 @@ def compute_lagrangian_descriptor(grid_parameters, vector_field, tau, p_value=0.
     LD : ndarray, shape (Nx, Ny)
         Array of computed Lagrangian descriptor values for all initial conditions.
     """
+    #get visualisation slice parameters and Number of DoF
+    if type(grid_parameters) == dict:
+        #n-DoF systems
+        slice_parameters = grid_parameters['slice_parameters'] # 2n-D grid
+        N_dof = len(grid_parameters['dof_slice'])
+    else:
+        #1-DoF systems
+        slice_parameters = grid_parameters # 2-D grid
+        N_dof = len(slice_parameters)
+        
+    #set boundaries for escape-box condition, if not defined
+    if not box_boundaries:
+        box_boundaries = int(N_dof/2)*[[-np.infty, np.infty]] #restricted to configuration space
```

Comment:
Would it be possible to avoid calling `check_if_points_escape_box` if `box_boundaries==False` instead of setting limits to infinity? This may be adressed as a separate issue.

Reply:
Provisionally, `box_boundaries` is passed to `vector_field_flat` via `compute_lagrangrian_descriptor`, as a way to work out the phase space dimensions and needed to reshape the input `u` points (flat 1D array) and use `check_if_points_escape_box` which only accepts ndarrays. 

As a way to avoid introducing a new variable. 

This can be enhanced in a future version.

Reply:
Added as a separate issue